tools/keycloak-user-initializer/src/main/java/ca/bc/gov/open/jag/tco/keycloakuserinitializer/controller/KeycloakInitializerController.java

```diff
@@ -0,0 +1,56 @@
+package ca.bc.gov.open.jag.tco.keycloakuserinitializer.controller;
+
+import java.util.List;
+
+import org.keycloak.representations.idm.UserRepresentation;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import ca.bc.gov.open.jag.tco.keycloakuserinitializer.idir.api.model.EnvironmentIdirUsersGet200ResponseDataInner;
+import ca.bc.gov.open.jag.tco.keycloakuserinitializer.service.KeycloakService;
+
+@RestController
+@RequestMapping(path = "api/user")
+public class KeycloakInitializerController {
+	
+	@Autowired
+    KeycloakService service;
+	
+	@GetMapping("/ping")
```

Comment:
I might suggest instead to just use Spring Boot's built-in Actuator endpoints. This way you could tie into the health check as well as other useful endpoints.

`<dependency>
    <groupId>org.springframework.boot</groupId>
    <artifactId>spring-boot-starter-actuator</artifactId>
</dependency>`